libafl_qemu/librasan/asan/Cargo.toml

```diff
@@ -21,6 +22,7 @@ default = [
 ]
 dlmalloc = ["dep:dlmalloc"]
 guest = []
+heap = []
```

Comment:
can you add a comment what it does?

Reply:
shouldn't the name be something like "other_allocator"?

Reply:
How about global_allocator?

Reply:
it's good

Reply:
or custom_allocator?

Reply:
I went with `global_allocator` in the end as it was probably the most directly descriptive (e.g. it enables a `#[global_allocator]`). In a `no_std` environment, there is no default allocator to customize. Hope that's ok.

Reply:
> can you add a comment what it does?

Added